data_prep/reuters_graph.py

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
Same as dataset.py except this will be completely gone

Reply:
Indeed